module.json

```diff
@@ -2,7 +2,7 @@
   "name": "forge-compendium-browser",
   "title": "The Forge: Compendium Library",
   "description": "Browse your compendium books like they were meant to be viewed. The Forge's Compendium Library lets you navigate your D&amp;D Beyond imported books in a beautiful and easy to use interface!",
-  "version": "1.2.2",
+  "version": "2.0",
```

Comment:
Don't switch from a `major.minor.patch` (semver) versioning system to a `major.minor` versioning system.